### DISCUSSION TITLE
Windows (cmd.exe) not recognizing ANSI codes with the print command

### DISCUSSION THREAD

**Discussion by acikek** [jotaen/klog#382] [Bugs]
Say I have this sample file:

```klg
2021-03-25
Back to #school
  8:30 - 11:30 Classes
  -10m Passing Period
  1h #homework
```

When I use `klog print` in a VSCode PowerShell terminal, it appears how I assume it should:

![image](https://user-images.githubusercontent.com/26909391/112566627-fafd7680-8da4-11eb-9140-ab073a324ac6.png)

However, when I run the same command in the Windows command prompt, it outputs:

![image](https://user-images.githubusercontent.com/26909391/112566688-11a3cd80-8da5-11eb-8f97-52f26de176d6.png)

I don't know if this is an issue on my end - cmd.exe [seems to have TrueColor support](https://gist.github.com/XVilka/8346728#now-supporting-true-color) - or if there isn't a platform-specific check for ANSI codes.

Additionally, it yields the same results in a Windows-native PowerShell terminal:

![image](https://user-images.githubusercontent.com/26909391/112566816-4ca60100-8da5-11eb-9561-de8bf6ee2747.png)

I browsed the issue board to see if anyone else has had this. Sorry in advance if this is a duplicate, I just want to bring this to attention.

**Comment by jotaen** [comment]
Yeah, that’s a known limitation at the moment, see the Windows install instructions in the [README](https://github.com/jotaen/klog), so the colours only work in Windows Terminal at the moment.

But in the end, I think it would be best if it just worked on all terminals on Windows, so it’s good to fix it anyways. (So let’s leave the ticket open.)

**Comment by vladdeSV** [comment]
I would like to add that even when using Windows Terminal, I get botched output when using *[Git for Windows](https://gitforwindows.org/)*.

![klog ouput gitbash](https://user-images.githubusercontent.com/7542961/112607178-376aba00-8e19-11eb-8327-b7a0c4be5451.gif)

**Comment by acikek** [comment]
Confirming that Windows Terminal works fine on my end.
